api/test/e2e/route_export_test.go

```diff
@@ -26,6 +26,24 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestExport_Route_Data_Empty(t *testing.T) {
+	tests := []HttpTestCase{
```

Comment:
is it necessary to confirm whether the route list is empty?

Reply:
I think this test returns an error when the route list is empty. Before this test, make sure whether the route list is empty will be repeated.